### DISCUSSION TITLE
Issue with Running edgex with security options

### DISCUSSION THREAD

**Discussion by SudheeM** [edgexfoundry/community#320] [Q&A]
Hardware used - ROM5620
Linux version 5.4.70-2.3.0 (Custom built)
Docker version 27.3.1, build ce12230 (https://github.com/docker/compose)
edgex - (https://github.com/docker/compose)

I'm trying run edgex on our platform and I have 2 different behavior

1. If I use 'docker-compose-no-secty-arm64' to start the edgex , it works fine. All the services are constantly running - No isssue with performance over period of 24hours
2. if I use  'docker-compose-arm64' to start the edgex , it does not seems to working correctly.  many of the services constantly re-starting, and over the period of 10-12 hours, system becomes slow. Any linux command takes considerable amount of time for the response

Any one come accross this kind of scenario ?  
What would be the best way to start resolving this issue ? 
Any thing obvious that i need to check 

Any input much appreciated!

**Comment by cloudxxx8** [comment]
you have to check the logs to see why thoe services were restarted.
also, you may enable the Core Data retention policy to prevent run out of the memory
https://docs.edgexfoundry.org/3.1/microservices/core/data/details/DataRetention/

It's disabled in v3.1 by default, and enabled in v4.0 (not released yet)
